vtex/admin-promotions
# Fix recurrency settings retrieval

#### What is the purpose of this pull request?
Fix the recurrency settings retrieval for saved promotions.

#### What problem is this solving?
Although the settings were being saved, the UI was incorrectly reading them, showing no recurrency settings after reload. 

#### How should this be manually tested?
Create a new promotion or edit an existing one adding custom recurrency settings. The settings should be preserved after refreshing page. 
https://vlauxexp--pricingqa.myvtex.com/admin/promotions/ 

#### Types of changes
